k8s/gitlab/log-cleaner.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: log-cleaner-script
+  namespace: gitlab
+data:
+  script: |
+    #! /bin/bash
+    set +o history
+    mc alias set gitlab https://minio.gitlab.spack.io \
+        "$ACCESS_KEY" "$SECRET_KEY"
+    set -o history
+    echo
+
+    mc find gitlab/gitlab-artifacts --maxdepth 5 --path '*_*_*' -exec \
+        'mc find {} --maxdepth 3 --older-than 7d --name job.log' \
```

Comment:
Seven days seems a little short, at least to me.  Since we have gitlab set to keep artifacts around for 30 days, maybe we could go with the same thing here?

Reply:
good call, I agree that we should pick an expiration horizon and stick with it